torch/_inductor/codegen/wrapper.py

```diff
@@ -1395,13 +1408,13 @@ def codegen_conditional(self, conditional):
         # TODO(aakhundov): make this work for C++ wrapper codegen (and ABI mode)
         self.writeline(f"{name} = [None] * {len(conditional.outputs)}")
         self.writeline(f"if {conditional.predicate.codegen_reference()}.item():")
-        self.writeline(IndentLine())
+        self.writeline(EnterScopeLine())
         self.codegen_subgraph(conditional.true_subgraph, outer_inputs, outer_outputs)
```

Comment:
Question: Is there any benefit from intermingling the codegen of subgraph with the parent graph? Could we instead codegen  the subgraph independently and treat that function as an opaque operator in the parent graph?

Reply:
Great question! Codegening subgraphs in separate functions is, indeed, an alternative. However, when deciding between the two alternatives, I've taken the inlining approach in https://github.com/pytorch/pytorch/pull/119759 mainly for two reasons:

1. To make the symbols defined in the main `call` function (e.g., the dynamic shape symbols) readily available in the nested subgraph codegen. Deriving dynamic shape symbols is not necessarily possible from the subgraph inputs, as those can have expressions in the shapes. So we'd have to pass dynamic shape symbols, too.

2. The current structure of the wrapper codegen and the way it's utilized in the client code seemed non-trivial to adapt to generating subgraphs as functions away from the current "cursor" in the wrapper.